presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueryFramework.java

```diff
@@ -295,6 +312,9 @@ protected void assertAccessDenied(
             }
             catch (RuntimeException e) {
                 assertExceptionMessage(sql, e, ".*Access Denied: " + exceptionsMessageRegExp);
+                if (exceptionsCauseMessageRegExp != null) {
+                    assertCauseExceptionMessage(sql, e, ".*Access Denied: " + exceptionsCauseMessageRegExp);
```

Comment:
Use AssertJ instead of something custom

Reply:
Sure! I thought it is not used in this class for some reason.